ksql-cli/src/test/java/io/confluent/ksql/CliTest.java

```diff
@@ -398,7 +397,8 @@ public void testSelectLimit() {
     for (int i = 1; i <= limit; i++) {
       GenericRow srcRow = streamData.get(Integer.toString(i));
       List<Object> columns = srcRow.getColumns();
-      GenericRow resultRow = new GenericRow(Arrays.asList(columns.get(1), columns.get(2)));
+      GenericRow resultRow = new GenericRow(Arrays.asList("\"" + columns.get(1) + "\"",
```

Comment:
why do we need the `"` ?

Reply:
This is the way JsonNode field return the string values. We can remove the `"`. As part of the struct work I am also syncing up with C3 team to come up with the final way of showing the results.

Reply:
Would that mean that when the result is displayed it will have a `"` ? Doesn't seem right if that is the case